docs/reference/sql/apis/sql-search-api.asciidoc

```diff
@@ -54,6 +54,11 @@ precedence.
 [[sql-search-api-request-body]]
 ==== {api-request-body-title}
 
+`allow_partial_search_results`::
+(Optional, Boolean) If `true`, returns partial results if there are shard
+request timeouts or shard failures. If `false`, returns an error with no
```

Comment:
I see this comment is slightly different in https://www.elastic.co/guide/en/elasticsearch/reference/current/search-search.html#search-search-api-query-params, ie. it has a link with `<<shard-failures,shard failures>>`.
Would it work here? Or maybe it's in a different module and it doesn't resolve the link?

Reply:
Link to shard failures now included.